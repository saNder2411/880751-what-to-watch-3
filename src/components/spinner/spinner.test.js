import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './spinner';


it(`Should Spinner render correctly`, () => {
  const markup = renderer
    .create(<Spinner />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
