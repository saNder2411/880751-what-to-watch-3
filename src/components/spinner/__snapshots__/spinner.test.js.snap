// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Spinner render correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="spinner__wrap"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
